README.md

```diff
@@ -41,7 +41,7 @@ tests in the `local_node-private` configuration.
 
 ## Docker
 
-Tests can be run with Docker with the following commands:
+Tests can be run with Docker with the following commands: (Note: These commands can be ../symlinked out of the repo with the local placeholder ./my_docker_run.sh file so it does not get checked in with your secrets)
```

Comment:
Took me a couple reads to understand what you mean here.

Can you provide explicit instructions for how to use the bash script?

E.g.
> 1. Copy the docker commands below into the file
> 2. Replace the fields with your secrets
> etc...

Reply:
Yeah, this was my late in the evening explanation, have now elaborated more. Let me know if it still not as clear as it should be.